src/utils/crypto.util.ts

```diff
@@ -53,6 +53,8 @@ export const encryptString$ = (value: string, password: string): Observable<Encr
   );
 };
 
+export const encryptPass$ = (password: string) => pbkdf2$(password, generateSalt());
```

Comment:
that's hashing, not encryption

Reply:
```
const pbkdf2$ = (password: BinaryLike, salt: BinaryLike) =>
  from<string>(NativeModules.Aes.pbkdf2(password, salt, 5000, 256));
```

too few rounds. According to the latest NIST recommendations and OWASP guidelines, for PBKDF2 with SHA-512, 120000 rounds should be used.

Reply:
@vixentael does `react-native-themis` provide hashing method, so we could get rid of `react-native-aes`?

Reply:
@letier3110 use `scrypt` instead of `pbkdf2`

Reply:
+1 to @unixvb